site/content/docs/02-template-syntax.md

```diff
@@ -643,6 +643,8 @@ If you're using `bind:` directives together with `on:` directives, the order tha
 
 Here we were binding to the value of a text input, which uses the `input` event. Bindings on other elements may use different events such as `change`.
 
+> Svelte bind attributes by JavaScript, so sometimes binding order is important. For example, `<input type="range" min="0" max="1" value={0.5} step="0.1"/>`, the value will be 1 because `step` is set after `value` is set. In this example, `value` should set at the last.
+
```

Comment:
Should this be in https://svelte.dev/docs#template-syntax-attributes-and-props as it's specific to the attribute order and not `bind:`?

```suggestion
> Sometimes, the attribute order matters as Svelte sets attributes sequentially in JavaScript. For example, `<input type="range" min="0" max="1" value={0.5} step="0.1"/>`, the value will be 1 because `step` is set after `value` is set. To fix it, `value` should set last.

```

Maybe something like this.

Reply:
updated! https://github.com/sveltejs/svelte/pull/7833/commits/d4e253adc24b03d69a806b65f95ce943eb55d90e

Yeah, your suggested section is better, so I moved it.

Reply:
I think "the value is rounded to `1` because `step` defaults to `1`" would be a tick clearer, but thanks a lot for the PR!

Reply:
THank you. updated! https://github.com/sveltejs/svelte/pull/7833/commits/6ee6c84ea3168bb47325e6611ae6db3a82ebfe9d